flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroRowDataDeserializationSchema.java

```diff
@@ -85,9 +88,27 @@ public AvroRowDataDeserializationSchema(
 			DeserializationSchema<GenericRecord> nestedSchema,
 			AvroToRowDataConverters.AvroToRowDataConverter runtimeConverter,
 			TypeInformation<RowData> typeInfo) {
+		this(nestedSchema, runtimeConverter, typeInfo, false);
+	}
+
+	/**
+	 * Creates a Avro deserialization schema for the given logical type.
+	 *
+	 * @param nestedSchema     Deserialization schema to deserialize as {@link GenericRecord}
+	 * @param runtimeConverter Converter that transforms a {@link GenericRecord} into {@link RowData}
+	 * @param typeInfo         The TypeInformation to be used by
+	 *                         {@link AvroRowDataDeserializationSchema#getProducedType()}
+	 * @param ignoreParseError Indicate whether to skip rows with parsing error
+	 */
+	public AvroRowDataDeserializationSchema(
```

Comment:
I would suggest to refactor this class into builder pattern. It's hard to maintain to introduce a new constructor when adding a new parameter.

Reply:
Besides, this class should be `@Internal` and not `@PublicEvolving`.